client/components/survey-container/components/survey-flow-card.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback } from '@wordpress/element';
+import classNames from 'classnames';
+import FlowCard from 'calypso/landing/stepper/declarative-flow/internals/steps-repository/components/flow-card';
+import { Option } from '../types';
+import { QuestionSelectionComponentProps } from './question-step';
+
+const SurveyFlowCard = ( {
+	onChange,
+	question,
+	value,
+	disabled,
+	onContinue,
+}: QuestionSelectionComponentProps ) => {
+	const questionKey = question?.key;
+
+	const handleClick = useCallback(
+		( questionKey: string, newValue: string[] ) => {
+			if ( ! value.includes( newValue[ 0 ] ) ) {
+				onChange( questionKey, newValue );
+			}
+		},
+		[ onChange, value ]
+	);
+
+	return (
+		<div
+			className={ classNames( 'question-flow-cards', {
+				'is-disabled': disabled,
+			} ) }
+		>
+			{ question?.options?.map( ( question: Option ) => {
+				return (
+					<FlowCard
+						key={ question.value }
+						title={ question.label }
+						text={ question.helpText ?? '' }
+						onClick={ () => {
+							if ( ! disabled ) {
+								handleClick( questionKey, [ question.value ] );
+								onContinue?.();
+							}
+						} }
+						icon={ undefined }
```

Comment:
We should probably pass the `disabled` value to the card itself so it doesn't call `onClick`. It's important for accessibility that the card knows it is disabled.

Reply:
Added the disabled attr to the FlowCard component and added to Card component itself.

Reply:
So we won't need the disabled check onClick since onClick won't be called when the card is disabled right?

Reply:
@heavyweight - Correct, I removed the "disabled" check inside the onClick function. The onClick won't be triggered if the component is disabled.